cmake/cmake_celix/ContainerPackaging.cmake

```diff
@@ -515,13 +527,17 @@ Optional Arguments:
 - LEVEL: The run level for the added bundles. Default is 3.
 - INSTALL: If this option is present, the bundles will only be installed instead of the default install and start.
            The bundles will be installed after all bundle in LEVEL 0..6 are installed and started.
+- COPY: If this option is present, the bundles will be copied to the container build dir. This option overrides the
```

Comment:
If both are present?

Reply:
Then COPY will be applied, but I will update this in generating an error because both should not be configured.